.changeset/tasty-spiders-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/mongo-nav': minor
```

Comment:
Hmm... this could probably be a patch, but I don't feel all that strongly about it.

Reply:
👍